website/docs/components/form/primitives/partials/code/component-api.md

```diff
@@ -34,6 +34,29 @@
   </C.Property>
 </Doc::ComponentApi>
 
+### Form::CharacterCount
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="controlId" @type="string">
+    The ID of the form control associated with the helper text. This is used to populate the `id` HTML attribute (with a `character-count-` prefix) of the element. This CharacterCount ID can then be referenced in the `aria-describedby` attribute of the form control.
```

Comment:
```suggestion
    The ID of the form control associated with the helper text. This is used to populate the element’s `id` HTML attribute (with a `character-count-` prefix). This CharacterCount ID can then be referenced in the `aria-describedby` attribute of the form control.
```

Reply:
For consistency, introducing this change requires changes to all the other form primitives.